core/controllers/incoming_emails.py

```diff
@@ -30,15 +28,14 @@ class IncomingReplyEmailHandler(base.BaseHandler):
     @acl_decorators.open_access
     def post(self, reply_to_id):
         incoming_mail = mail.InboundEmailMessage(self.request.body)
-        model = (
-            email_models.GeneralFeedbackEmailReplyToIdModel
-            .get_by_reply_to_id(reply_to_id))
+        feedbackReplyToId = email_services.get_general_feedback_reply_to_id(
```

Comment:
Use snake_case for var names.

@apb7 @lilithxxx surprised the linter doesn't catch this? (Is the variable regex correct in .pylintrc?)

Reply:
created #5943 for this